packages/linter/src/builders/lint/lint.impl.ts

```diff
@@ -31,14 +376,4 @@ function run(options: any, context: BuilderContext): Observable<BuilderOutput> {
   );
 }
 
-// remove once https://github.com/angular/angular-cli/issues/15053 is fixed
-function patchedLogger(context: any): any {
-  const s = context.logger._subject.next;
-  context.logger._subject.next = (v: any) => {
-    v.message = v.message.replace('<???>', context.target.project);
-    return s.apply(context.logger._subject, [v]);
-  };
-  return context.logger;
-}
```

Comment:
@FrozenPandaz - I saw you created the original issue in the comment above - it seems to have been fixed - we should be okay to remove the above, right?